gc-cli/bin/gc-cli.js

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env node
 "use strict";
 
+require("babel-register");
```

Comment:
If I add `ignore: false` when I use the script to convert any library I get the following stacktrace:
```
[BABEL] Note: The code generator has deoptimised the styling of "C:/dev/gc-cli/node_modules/immutable/dist/immutable.js" as it exceeds the max of "100KB".
[BABEL] Note: The code generator has deoptimised the styling of "C:/dev/gc-cli/node_modules/lodash/dist/lodash.js" as it exceeds the max of "100KB".
C:\dev\gc-cli\node_modules\babel-core\lib\transformation\file\index.js:556
      throw err;
      ^

SyntaxError: C:/dev/gc-cli/node_modules/globule/node_modules/graceful-fs/graceful-fs.js: Invalid number (80:53)
  78 |     switch (req.constructor.name) {
  79 |       case 'OpenReq':
> 80 |         open(req.path, req.flags || "r", req.mode || 0777, req.cb)
     |                                                      ^
  81 |         break
  82 |       case 'ReaddirReq':
  83 |         readdir(req.path, req.cb)
    at Parser.pp.raise (C:\dev\gc-cli\node_modules\babylon\index.js:1378:13)
    at Parser.readNumber (C:\dev\gc-cli\node_modules\babylon\index.js:5365:12)
    at Parser.getTokenFromCode (C:\dev\gc-cli\node_modules\babylon\index.js:5205:21)
    at Parser.readToken (C:\dev\gc-cli\node_modules\babylon\index.js:4887:19)
    at Parser.nextToken (C:\dev\gc-cli\node_modules\babylon\index.js:4877:19)
    at Parser.next (C:\dev\gc-cli\node_modules\babylon\index.js:4802:10)
    at Parser.pp.parseExprOp (C:\dev\gc-cli\node_modules\babylon\index.js:442:12)
    at Parser.pp.parseExprOps (C:\dev\gc-cli\node_modules\babylon\index.js:419:17)
    at Parser.pp.parseMaybeConditional (C:\dev\gc-cli\node_modules\babylon\index.js:397:19)
    at Parser.pp.parseMaybeAssign (C:\dev\gc-cli\node_modules\babylon\index.js:360:19)
```